Makefile

```diff
@@ -72,7 +72,7 @@ DIST_OS_TAG ?= $(BASE_OS_TAG)
 ifeq ($(BASE_OS),ubuntu)
   BASE_OS_TAG=$(BASE_OS_TAG_UBUNTU)
   BASE_IMAGE ?= ubuntu:$(BASE_OS_TAG_UBUNTU)
-  INSTALL_DRIVER_VERSION ?= "21.48.21782"
+  INSTALL_DRIVER_VERSION ?= "22.35.24055"
```

Comment:
Shouldn't we do the same for redhat?

Reply:
the latest rh package is now 22.28

Reply:
Yes, but in the section below (line 82 of this file) we have INSTALL_DRIVER_VERSION ?= "21.38.21026". 
Should we update it to 22.28?